AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx

```diff
@@ -30,10 +30,8 @@ export default class SearchResult extends React.Component {
 
     renderResultBody(result) {
         return Object.keys(result).map(key => {
-            if (result[key] == null || result[key] == '')
-            {return null;}
-            if (key == 'reportTitle')
-            {return null;}
+            if (result[key] == null || result[key] == ''){return null;}
+            if (key == 'reportTitle'){return null;}
```

Comment:
Is this lint formatting the code wrong? You might need to manually undo this formatting, and have it as:
```suggestion
            if (result[key] == null || result[key] == ''){
                return null;
            }
            if (key == 'reportTitle'){
                return null;
            }
```